sdk/src/main/java/io/radar/sdk/RadarBeaconManager.kt

```diff
@@ -617,12 +599,22 @@ internal class RadarBeaconManager(
         }
     }
 
-    internal fun isBluetoothSupported(context: Context): Boolean {
+    private fun isBluetoothSupported(context: Context): Boolean {
         if (!this::adapter.isInitialized) {
             adapter = BluetoothAdapter.getDefaultAdapter()
         }
 
         return context.packageManager.hasSystemFeature(PackageManager.FEATURE_BLUETOOTH) && adapter != null && adapter.bluetoothLeScanner != null
     }
 
+    private fun getScanSettings(): ScanSettings {
```

Comment:
I could see wanting to put these in remote tracking options or config and changing the values. But not something to do right now.